manifests/v/ViviCorporation/Vivi/3.3.6/ViviCorporation.Vivi.installer.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ViviCorporation.Vivi
+PackageVersion: 3.3.6
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
+UpgradeBehavior: install
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://downloads.vivi.io/app/3.3.6/ViviSetup64.msi
+  InstallerSha256: AFD39113B01F6710D9F5AF3FC986E9DBCB6335DA173DF50596D3C4BE5A894965
+  ProductCode: '{A25E5188-3C67-4D72-91B0-E78083FF34CF}'
+- InstallerLocale: en-US
+  Architecture: neutral

```

```suggestion
  Architecture: x86
```
I believe neutral covers all architectures (including arm/arm64); better to keep it as x86 here.